src/systemctl/systemctl-logind.c

```diff
@@ -339,9 +330,7 @@ int logind_schedule_shutdown(void) {
                 return log_warning_errno(r, "Failed to call ScheduleShutdown in logind, proceeding with immediate shutdown: %s", bus_error_message(&error, r));
 
         if (!arg_quiet)
-                log_info("%s scheduled for %s, use 'shutdown -c' to cancel.",
-                         log_action,
-                         FORMAT_TIMESTAMP_STYLE(arg_when, arg_timestamp_style));
+                logind_show_shutdown();
```

Comment:
I do not think it is not necessary to retrieve the `ScheduledShutdown` property here.
How about to introduce tiny helper `log_scheduled_shutdown()` instead?

Reply:
reading back the property makes sure we actually print the server side view though.

Reply:
Yeah, I think this makes things simpler and easier.